group.yml

```diff
@@ -29,7 +29,7 @@ non_release:
   - kuryr-controller
   - ose-leader-elector
   - ose-manila-provisioner
-  - ose-ovn-kubernetes
+  - ovn-kubernetes
```

Comment:
This refers to the dist-git changing. Are you sure you want to request a new dist-git for this change? (would have to wait until it exists to merge this) The file name change below matches this, but I'm not sure there's any actual reason to change any of the internal entities just to start using the public upstream repo. We build lots of public things that internally add a ose- prefix.

Reply:
/hold

This would break the references.  Your name has to be ose-* to be in the payload.